src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public java.lang.String toString() {
```

Comment:
Use `name()` for getting String representation of enum constants. If the `color` field is meant to be an enum, you should use `color.name()` instead of directly using `color` in the `toString` method. This ensures that you get the correct String representation of the enum constant.